nss-day-cohort-46/rare-server-tragic-trolls
# jhf-check-subscribed

# Description
made a check_subscribed function that returns {subscribed:true/false} depending on whether or not the user logged in is subscribed or not
## Type of change



# Testing Instructions
git fetch all
checkout to jhf-check-subscribed on server
checkout to jhf-user-detail on client side

log in to client go to users, click a user detail, the subscribe button should be functional

# Checklist:
